website/client/src/components/shared/closeIcon.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
thanks for this! We should probably change all modals to use this, maybe in the next pr?

Reply:
after this is merged, I don't see a reason not to do it^^